### DISCUSSION TITLE
线上构建失败 Cannot find module 'umi-plugin-keep-alive' from '/var/jenkins_home/workspace/***'

### DISCUSSION THREAD

**Discussion by Cooper1886** [umijs/umi#12340] [Q&A]
有人也遇到过吗
场景：umi3 安装umi-plugin-keep-alive 
问题：本地使用和build都没有问题  线上构建的时候报错找不到包
Cannot find module 'umi-plugin-keep-alive' from '/var/jenkins_home/workspace/***'
Error: Cannot find module 'umi-plugin-keep-alive' from '/var/jenkins_home/workspace/***
    at Function.resolveSync [as sync] (/var/jenkins_home/workspace/***/node_modules/@umijs/deps/compiled/resolve/index.js:1:11170)

**Comment by fz6m** [comment]
umi 3 不维护了，建议升级到 umi 4
